docs/ethical-advertising.rst

```diff
@@ -4,13 +4,26 @@ Ethical Advertising
 Read the Docs is a large,
 free web service.
 There is one proven business model to support this kind of site:
-Advertising.
-We're building an advertising model that respects users while providing value to advertisers.
+**Advertising**.
+I know it's a dirty word,
```

Comment:
This is great feedback. I think it's definitely counter-productive to talk negatively about the thing we've settled on. I'm just trying to channel the developer mindset about ads, but this is another way that you can tell I'm not a writer :)